src/languages/es.ts

```diff
@@ -3771,5 +3772,24 @@ export default {
             security: 'Expensify es PCI-DSS obediente, utiliza cifrado a nivel bancario, y emplea infraestructura redundante para proteger tus datos.',
             learnMoreAboutSecurity: 'Conozca más sobre nuestra seguridad.',
         },
+        subscriptionSettings: {
+            title: 'Configuración de suscripción',
+            autoRenew: 'Auto-renovación',
```

Comment:
I see that most of the mocks say `Auto-increase annual seats` but not all, at least one says `Auto-increase` only. So we either need to get an updated translation for `Auto-increase annual seats`, or update the English text to be only `Auto-increase`.

@MitchExpensify or maybe @dannymcclain, do you know which text we're supposed to go with?

Reply:
I think `Auto-increase annual seats` is correct, but I'd like @MitchExpensify to confirm.

Reply:
Is this the toggle label in the card on the `Subscription` page? If so, I think `Auto-increase` is fine. Especially since: 

1. We also add the green text next to it, so it makes the header longer 
2. The first sentence of the explainer underneath the toggle pretty much says that as well verbatim 

<img width="505" alt="image" src="https://github.com/Expensify/App/assets/16232057/9c9ec12d-4aea-4a7e-b36b-65e2c86f5d33">

Reply:
Oh good call. That works for me!